scripts/init_e2e.sh

```diff
@@ -61,5 +61,10 @@ sleep_until_next_epoch
 lavad tx project del-keys -y "$user3addr-myproject" --from user3 cookbook/projects/example_project_keys.yml --gas-prices=$GASPRICE
 sleep_until_next_epoch
 
+lavad tx subscription del-project myproject -y --from user3 --gas-adjustment "1.5" --gas "auto" --gas-prices $GASPRICE
+sleep_until_next_epoch
+
+lavad q subscription list-projects user3
+
 # we need to wait for the next epoch for the stake to take action.
 sleep_until_next_epoch
```

Comment:
we dont need another sleep, this is just a query

Reply:
true - this was here before ... removed now.